doc/source/whatsnew/v2.0.0.rst

```diff
@@ -1423,27 +1406,21 @@ ExtensionArray
 Styler
 ^^^^^^
 - Fix :meth:`~pandas.io.formats.style.Styler.background_gradient` for nullable dtype :class:`Series` with ``NA`` values (:issue:`50712`)
--
 
 Metadata
 ^^^^^^^^
 - Fixed metadata propagation in :meth:`DataFrame.corr` and :meth:`DataFrame.cov` (:issue:`28283`)
--
 
 Other
 ^^^^^
 
 - Bug in :meth:`Series.searchsorted` inconsistent behavior when accepting :class:`DataFrame` as parameter ``value`` (:issue:`49620`)
 - Bug in :func:`array` failing to raise on :class:`DataFrame` inputs (:issue:`51167`)
--
-
-.. ***DO NOT USE THIS SECTION***
-
--
--
 
 .. ---------------------------------------------------------------------------
 .. _whatsnew_200.contributors:
 
 Contributors
 ~~~~~~~~~~~~
+
+.. contributors:: v1.5.0..v2.0.0|HEAD
```

Comment:
This should be 1.5.3..?

That's what we did in 1.5.0 release notes

Reply:
Good point. I saw it, but I think it's wrong. 1.5.3 is in a separate branch. The contributors for 2.0.0 are the authors of commits from 1.5.0rc0 to 2.0.0 (v1.5.0 is wrong also, thanks for letting me think about this again). I'm not sure how git will resolve the commits from 1.5.3 to 2.0.0 since there is no path in the git graph between those versions. Maybe it's equivalent, and what it does is to return the commits since the branches diverged. But I think using v1.5.0rc0..v2.0.0 is more correct. Does it make sense to you?

Reply:
Hm makes sense, but have never look at the actual script extracting the contributors

Reply:
The contributors are obtained by checking the authors from something like `git log --oneline -s v1.5.3..v2.0.0rc0`, which indeed, seems to ignore all the commits in the `1.5.x` branch, and return the commits since branches diverged until the end tag. I think using the correct tag is still better to not mislead, or in case git ever decides to raise instead of returning the commits since branches diverged.